include/rmm/mr/device/tracking_resource_adaptor.hpp

```diff
@@ -65,6 +65,47 @@ class tracking_resource_adaptor final : public device_memory_resource {
         allocation_size{size} {};
   };
 
+  /**
```

Comment:
This class should be in the `detail` namespace.

Reply:
Even if its inside the class declaration? If so, should I move the existing struct `allocation_info` as well?

Reply:
Nevermind, Github doesn't give me full context by default.